po/de.po

```diff
@@ -548,6 +563,12 @@ msgstr "Audio-Gruppe %d"
 msgid "Audio group name"
 msgstr "Neuer Audio-Gruppenname"
 
+msgid "Audio recording:"
+msgstr "Audio-Aufnahme:"
+
+msgid "Auto add new devices"
+msgstr "Neue Geräte automatisch hinzufügen"
```

Comment:
It seems too long. Can you make it shorter?

Reply:
It actually still fits well with the smallest window width.

![Bildschirmfoto von 2022-01-26 20-15-59](https://user-images.githubusercontent.com/6174457/151231208-544749c0-8a15-46da-9ef3-0c751d2b401d.png)